openmdao/components/balance_comp.py

```diff
@@ -21,8 +21,8 @@ def __init__(self, name=None, eq_units=None, lhs_name=None,
         r"""
         Initialize a BalanceComp, optionally creating a new implicit state variable.
 
-        The BalanceComp is a bit like IndepVarComp in that it allows for the
-        creation of one or more implicit state variables, and computes the residuals
+        The BalanceComp is a bit like IndepVarComp in that it allows for the \
```

Comment:
are these end of line '\'s actually necessary?

Reply:
Oh, no, you're right, that was an attempt to get rid of the sphinx warnings, based on other people's reports on the internet. I'll pull those out, they had zero effect.

Reply:
Fixed.